File: beginner/javascript/1006.js
var input = require('fs').readFileSync('stdin', 'utf8');

var values = input.split('\n');

var nota1 = parseFloat(values.shift());
var nota2 = parseFloat(values.shift());
var nota3 = parseFloat(values.shift());

var media = ((nota1 * 2.0) + (nota2 * 3.0) + (nota3 * 5.0)) / 10;

console.log("MEDIA = " + media.toFixed(1));

File: beginner/c/1006.c
#include <stdio.h>

int main(){

    double nota1;
    double nota2;
    double nota3;

    scanf("%lf", &nota1);
    scanf("%lf", &nota2);
    scanf("%lf", &nota3);

    double media = ((nota1 * 2.0) + (nota2 * 3.0) + (nota3 * 5.0)) / 10;

    printf("MEDIA = %.1lf\n", media);

    return 0;
}
